IanGRap/Squish
# Taunt button

Contributors: Alfredo Cuevas

Goal: Remap the Y button to be the taunt / dab.

What Changed: 
* PlayerInput.cs: Now saves Y button press into the PressTaunt variable
* (New) PlayerTaunt.cs: Activates taunt and keeps a timer for the time taunting
* PlayerJump.cs: Added a check to prevent jumping while taunting
* PlayerMovement.cs: Added a check to stop movement/ prevent movement while taunting
* PlayerPrefab: I added the script to the prefab

How to Test: 
* Load any Scene with movable players. I used First Release. Also test in Lobby.
* Press Y to activate taunt
* Make sure there aren't any weird interactions with taunting by
  * Try moving around and taunting
  * Try jumping and taunting
  * Try dashing and taunting
